warehouse/integrations/redshift/redshift.go

```diff
@@ -255,15 +255,32 @@ func (rs *Redshift) AddColumns(tableName string, columnsInfo []warehouseutils.Co
 			columnInfo.Name,
 			getRSDataType(columnInfo.Type),
 		)
-		rs.Logger.Infof("AZ: Adding column for destinationID: %s, tableName: %s with query: %v", rs.Warehouse.Destination.ID, tableName, query)
+		rs.Logger.Infof("RS: Adding column for destinationID: %s, tableName: %s with query: %v", rs.Warehouse.Destination.ID, tableName, query)
 
 		if _, err := rs.DB.Exec(query); err != nil {
+			if CheckAndIgnoreColumnAlreadyExistError(err) {
+				pkgLogger.Infof("RS: Column already exists for destinationID: %s, tableName: %s with query: %v", rs.Warehouse.Destination.ID, tableName, query)
```

Comment:
We would need to suppress the errors in case of a single column alter since Redshift does support ADD COLUMN IF NOT EXISTS